### PR TITLE
Add jsblocks

### PR DESCRIPTION
**Pull request by astoilkov**

### CODE REVIEW

File: files/blocks/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "blocks",
+  "repo": "astoilkov/jsblocks",
+  "files": {
+    "basePath": "dist/"
+  }
```
Comment (jimaek) [inline files/blocks/update.json]: add also an include parameter for:
"*.min.*", "./mvc/*.min.*", "./query/*.min.*"

Comment (astoilkov) [inline files/blocks/update.json]: Is this going to include all files in all directories. Including the .js and .map files in `mvc` and `query` folders?

Comment (jimaek) [inline files/blocks/update.json]: It will include only the min files from dist, mvc and query

Comment (astoilkov) [inline files/blocks/update.json]: Can we do:
```javascript
{
  "packageManager": "github",
  "name": "blocks",
  "repo": "astoilkov/jsblocks",
  "files": {
    "include": ["*.min.*", "./mvc/*.min.*", "./query/*.min.*", "*.map", "./mvc/*.map", "./query/*.map", "./mvc/*.js", "./query/*.js"],
    "basePath": "dist/"
  }
}
```

Comment (jimaek) [inline files/blocks/update.json]: Why do you want the unminified files?

Comment (astoilkov) [inline files/blocks/update.json]: So people can take a look at the source code. Howerver, it is not that important. I could leave them out if you suggest to do so.

Comment (jimaek) [inline files/blocks/update.json]: Well, jsDelivr is for production websites. Source code is not relevant there, the most important is speed. My original config should work fine for that if you agree

**Comment by astoilkov** [comment]
I agree with you. You could take a look at the final version.

**Pull request merged**